internal/testrunner/runners/pipeline/runner.go

```diff
@@ -33,9 +34,15 @@ const (
 	TestType testrunner.TestType = "pipeline"
 )
 
+var (
+	serverlessEnableCompareResults = environment.WithElasticPackagePrefix("SERVERLESS_PIPELINE_TEST_ENABLE_COMPARE_RESULTS")
```

Comment:
Add somewhere a TODO comment mentioning that this is a temporary workaround till we have something better for deterministic geoip in serverless.

Reply:
Updated the TODO message here: https://github.com/elastic/elastic-package/pull/1521/commits/6b35523e5203354b5b6db1304031eece5bd21967